hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -8,11 +8,51 @@
 
   datawidth: "32",  # 32-bit datawidth
 
-  clocks: [
-    { name: "main", freq: "100000000" }
-    { name: "fixed", freq: "100000000" }
-    { name: "usb", freq: "48000000" }
-  ]
+  // This is the clock data strcture of the design.
+  // The src key indicates the raw clock sources in the design
+  // The groups key indicates the various clock groupings in the design
+  clocks: {
+
+    srcs: [
+      { name: "main",      freq: "100000000" }
+      { name: "fixed",     freq: "100000000" }
+      { name: "usb_48mhz", freq: "48000000" }
```

Comment:
@eunchan 
@vogelpi 

this is renamed because i want to use the `src` name in directly constructing the net name during assignment. So in the future, I'm going to simplify this down to just `clk_usb`.  For this PR, I didn't want to make that change, but rather has part of a clock renaming PR.